components/collapsible-panel/collapsible-panel-summary-item.js

```diff
@@ -39,7 +52,13 @@ class CollapsiblePanelSummaryItem extends SkeletonMixin(LitElement) {
 	}
 
 	render() {
-		return html`<p class="d2l-body-small d2l-skeletize">${this.text}</p>`;
+		const classes = {
+			'd2l-body-small': true,
+			'd2l-skeletize': true,
+			'truncate': this.lines > 0
+		};
+		const styles = (this.lines > 0) ? { '-webkit-line-clamp': this.lines } : {};
```

Comment:
Why aren't we adding this rule in the `truncate` definition?

Reply:
Just because the value is coming from the consumer-specified attribute. We _could_ set a CSS variable in one of the life cycle methods to use in CSS, but seems unnecessary complexity for this one property. It would be amazing if we could just use [attr](https://developer.mozilla.org/en-US/docs/Web/CSS/attr), but it would not work in this context... maybe one day.